docs/tasks/20260429_1623_sankey-svgノードラベル常時表示設計.md

```
@@ -0,0 +1,122 @@
+
+---
+
+## 現状
+
+### `showLabels` の役割
+
+`showLabels`（URL param `sl`）は2つの箇所に影響している。
+
+**① レイアウト計算（`page.tsx` ~L772）**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)